pkg/tests/tasks/security/authorization/ext_auth_test.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2021 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package authorization
+
+import (
+	"testing"
+
+	"github.com/maistra/maistra-test-tool/pkg/app"
+	"github.com/maistra/maistra-test-tool/pkg/util/check/assert"
+	"github.com/maistra/maistra-test-tool/pkg/util/hack"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/pod"
+	"github.com/maistra/maistra-test-tool/pkg/util/retry"
+	"github.com/maistra/maistra-test-tool/pkg/util/test"
+)
+
+func TestExtAuth(t *testing.T) {
+	test.NewTest(t).Id("T37").Groups(test.Full, test.InterOp).Run(func(t test.TestHelper) {
+		hack.DisableLogrusForThisTest(t)
+
+		ns := "foo"
+		t.Cleanup(func() {
+			oc.DeleteFromString(t, ns, ExternalAuthzService)
+			oc.RecreateNamespace(t, ns)
+			oc.RecreateNamespace(t, meshNamespace)
+		})
+
+		t.Log("This test validates authorization policies with a JWT Token")
+
+		t.LogStep("Install httpbin and sleep")
+		app.InstallAndWaitReady(t, app.Httpbin(ns), app.Sleep(ns))
+
+		t.LogStep("Check if httpbin returns 200 OK when no authorization policies are in place")
+		verfiyHttpbinRequest(t, ns, httpbinRequest("GET", "/ip"))
```

Comment:
typo
```suggestion
		verifyHttpbinRequest(t, ns, httpbinRequest("GET", "/ip"))
```

Reply:
Also, rename this function so its name clearly states what it does. I don't know what "verify" means.

Reply:
httpbinRequestVerfication, If it is not good. 

Please suggest some name.

Reply:
If the function is performing an assertion, the name should clearly reflect that, so that when someone reads the test code, they can immediately identify where assertions are being made (as that's the most important part of the test).

Maybe call it `assertHttpbinRequestSucceeds` or `assertHttpbinReturns200`.